sdk/keyvault/Azure.Security.KeyVault.Certificates/CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
-# Release History
+Release History
+================
 
-## 4.0.0-preview.7
+# 4.0.0-preview.7
```

Comment:
A single `#` produces an H1, just like `===` so this will look wrong when rendered as HTML since both will be the same size. I think we should just make the title H1 and release numbers H2 and adjust the guidelines accordingly.

Reply:
I opened https://github.com/Azure/azure-sdk/pull/852 to settle all CHANGELOG.md differences.